compose/install/compose-linux.md

```diff
@@ -0,0 +1,120 @@
+---
+description: How to install Docker Compose on Linux
+keywords: compose, orchestration, install, installation, docker, documentation
+toc_max: 3
+
+title: Install on Linux
+redirect_from:
+- /compose/compose-plugin/
+---
+
+On this page you can find instructions on how to install the Compose on Linux from the command line.
+
+## Install Compose
+
+To install Compose:
+* Option 1: [Set up Docker's repository on your Linux system](#install-using-the-repository).
+* Option 2: [Install Compose manually](#install-the-plugin-manually).
+
+### Install using the repository
+
+> **Note**
+>
+> These instructions assume you already have Docker Engine and Docker CLI installed and now want to install the Compose plugin.  
+For Compose standalone, see [Install Compose Standalone](./compose-other.md#on-linux).
+
+If you have already set up the Docker repository, jump to step 2.
+
+1. Set up the repository in:
+[Ubuntu](../../engine/install/ubuntu.md/#set-up-the-repository) |
+[CentOS](../../engine/install/centos.md/#set-up-the-repository) |
+[Debian](../../engine/install/fedora.md/#set-up-the-repository) |
+[RHEL](../../engine/install/fedora.md/#set-up-the-repository) |
+[SLES](../../engine/install/sles.md/#set-up-the-repository).
+
+2. Update the `apt` package index, and install the _latest version_ of Docker Compose:
+
+    > Or, if using a different distro, use the equivalent package manager instructions.
+
+    ```console
+    $ sudo apt-get update
+    $ sudo apt-get install docker-compose-plugin
+    ```
+
+    Alternatively, to install a specific version of the Compose CLI plugin:
```

Comment:
I wonder into how much detail we want to go for installing older versions. I know we added the equivalent sections in the "engine" installation instructions for production scenarios (where enterprise users may want to "pin" to specific versions), but as this section already tries to be "generic" ("or if using a different distro"), perhaps we should keep the installation also generic ("install latest stable release"). In the end, "how to install a specific version of a package" is more of a general ["know how to work with your system's package manager"](https://linuxopsys.com/topics/install-specific-version-package-apt) topic.

Reply:
This is an important question, I would prefer to sync on this and probably save this for a second round of updates to page. As we will also have feedback from users soon.

Reply:
As discussed leave essential instructions for update, install and check version.